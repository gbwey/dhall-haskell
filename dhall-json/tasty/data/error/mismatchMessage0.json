{
    "a": {
        "b": [
            10,
            0.5
        ]
    },
    "c": {
        "d": [
            10,
            0.5
        ]
    }
}
