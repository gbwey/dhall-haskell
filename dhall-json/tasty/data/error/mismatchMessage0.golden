[1;31m
Error: [0m$.a.b[1]: Dhall type expression and JSON value do not match:

Expected Dhall type:
Natural

JSON:
0.5
